packages/client/src/v2-events/features/events/useEvents/procedures/actions/action.ts

```diff
@@ -225,6 +247,25 @@ export function useEventAction<P extends DecorateMutationProcedure<any>>(
       }
       const fields = getActiveActionFields(eventConfiguration, actionType)
 
+      const options: Record<string, unknown> = {}
+
+      if (callBacks) {
+        if (callBacks.onSuccess) {
+          options.onSuccess = async (updatedDocument: EventDocument) => {
+            callBacks.onSuccess?.(updatedDocument)
+            return updateLocalEvent(updatedDocument)
```

Comment:
This might not work correctly if you 

1. Create the event when you are offline / when a request fails
2. Refresh the browser so the mutation is fetched from IndexedDB

Reply:
These also override mutation defaults defined in the root of the file, which might not be what we want to do here (e.g. https://github.com/opencrvs/opencrvs-core/pull/8977/files#diff-c7438a7d33bdcc44b6fbbe68e3ae137337c11e9ae6fa571c6fea32a9f30ddf03L162)